store/tikv/coprocessor.go

```diff
@@ -441,12 +520,16 @@ type copIteratorWorker struct {
 
 // copIteratorTaskSender sends tasks to taskCh then wait for the workers to exit.
 type copIteratorTaskSender struct {
+	// newTaskCh is a channel with tasks we need to process
+	newTaskCh <-chan *copTask
+	// taskCh is a channel with tasks for a worker
 	taskCh   chan<- *copTask
 	wg       *sync.WaitGroup
-	tasks    []*copTask
 	finishCh <-chan struct{}
 	respChan chan<- *copResponse
 	sendRate *rateLimit
+	// tasks for copIterator.Next (if needed)
```

Comment:
Could you make it more specific, because I do not get it with my first look.

Reply:
Sure. Fixed